app/views/users/login.erb

```diff
@@ -0,0 +1,11 @@
+<h3> Login </h3>
+
+<div class="form-group">
+  <form method="post" action="/users/login">
+    <label for="email">Email:</label>
+    <input type="text" name="email">
```

Comment:
type on this should be "email". works in some browsers, particularly mobile @kmbradshaw